hermes_spani/data/configrc

```diff
@@ -22,34 +22,3 @@ time_format = %Y-%m-%d %H:%M:%S
 ; SunPy working directory.
 ; Default value: data/
 download_dir = data
-
```

Comment:
@ehsteve sorry I'm a little out of the loop so I'm just checking, why did you delete the logger?  Just curious

Reply:
Yes, but that is because we are using the `hermes_core` logger. The individual packages do not have their own logger. This config was left over from when each package managed their own logger.